core/src/ICryptographyManager.cs

```diff
@@ -32,6 +32,7 @@ internal interface ICryptographyManager
         string CreateBase64UrlEncodedSha256Hash(string input);
         string GenerateCodeVerifier();
         string CreateSha256Hash(string input);
+        byte[] CreateSha256HashBytes(string input);
```

Comment:
aside: we should also be supporting 512... possibly something to align with other libraries on afaik evo supports both right now.